climada/entity/impact_funcs/trop_cyclone.py

```diff
@@ -34,6 +35,135 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class CountryCode(Enum):
```

Comment:
Why does this enum contain numerical **and** alpha-3 country codes? Conversion between the two can be done by [`country_to_iso`](https://github.com/CLIMADA-project/climada_python/blob/08ec51bb958436cd18e77cf8f4178bc05a295d78/climada/util/coordinates.py#L1424) and this effectively doubles the functionality

Reply:
Sure, only one is enough, just thought it was better practice to have both in case one might need them, I can delate one or the other if you prefer.

Reply:
Ok, I will delate one then 👍